src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -139,6 +139,10 @@ public function getConfigTreeBuilder()
                                 }
 
                                 if (false !== strpos($v, '/')) {
+                                    if ('0.0.0.0/0' === $v) {
+                                        return false;
+                                    }
```

Comment:
That's probably true, but as the `Configuration` class won't be checked at runtime it doesn't really matter.